shared/src/testFixtures/kotlin/com/egm/stellio/shared/util/JsonLdContextUtils.kt

```diff
@@ -28,3 +28,5 @@ const val NAME_PROPERTY = "https://schema.org/name"
 const val CREATED_BY_RELATIONSHIP = "https://ontology.eglobalmark.com/egm#createdBy"
 const val MANAGED_BY_COMPACT_RELATIONSHIP = "managedBy"
 const val MANAGED_BY_RELATIONSHIP = "https://ontology.eglobalmark.com/egm#managedBy"
+const val DATE_OF_FIRST_BEE_COMPACT_PROPERTY = "dateOfFirstBee"
+const val DATE_OF_FIRST_BEE_PROPERTY = "https://uri.etsi.org/ngsi-ld/default-context/dateOfFirstBee"
```

Comment:
don't you think your URI does not look like the others? 🤔

Reply:
I can put this instead `const val DATE_OF_FIRST_BEE_PROPERTY = "https://uri.etsi.org/ngsi-ld/default-context/$DATE_OF_FIRST_BEE_COMPACT_PROPERTY"` but otherwise I don't see

Reply:
1. you are using the core context from NGSI-LD, this is a no-go zone for your own stuff
2. other terms are under `ontology.eglobalmark.com/...` (you'll probably have to add it in the context managed on GH, and to update the cached version in `shared/src/testFixtures/resources/contexts/apic.jsonld`)

Reply:
How I do to update the cached ?

Reply:
hmm, the response is the point 2 in my previous commit 🤔